core/src/main/scala/com/nvidia/spark/rapids/tool/analysis/AnalysisUtils.scala

```diff
@@ -34,3 +34,43 @@ object StageAccumDiagnosticMetrics {
     SW_TOTAL_BYTES_METRIC, SR_FETCH_WAIT_TIME_METRIC, SW_WRITE_TIME_METRIC,
     GPU_SEMAPHORE_WAIT_METRIC)
 }
+
+object IOAccumDiagnosticMetrics {
+  val OUTPUT_ROWS_METRIC = "output rows" // other names: join output rows, number of output rows
+  val SCAN_TIME_METRIC = "scan time"
+  val OUTPUT_BATCHES_METRIC = "output columnar batches"
+  val BUFFER_TIME_METRIC = "buffer time"
+  val SHUFFLE_WRITE_TIME_METRIC = "shuffle write time"
+  val FETCH_WAIT_TIME_METRIC = "fetch wait time"
+  val GPU_DECODE_TIME_METRIC = "GPU decode time"
+
+  /**
+   * Get all IO diagnostic metrics names
+   */
+  def getAllIODiagnosticMetrics: Set[String] = Set(
+    OUTPUT_ROWS_METRIC,
+    SCAN_TIME_METRIC,
+    OUTPUT_BATCHES_METRIC,
+    BUFFER_TIME_METRIC,
+    SHUFFLE_WRITE_TIME_METRIC,
+    FETCH_WAIT_TIME_METRIC,
+    GPU_DECODE_TIME_METRIC)
+
+  /**
+   * Check if a metric name belongs to IO diagnostic metrics
+   */
+  def isIODiagnosticMetricName(metric: String): Boolean = {
+    getAllIODiagnosticMetrics.contains(metric) || metric.contains(OUTPUT_ROWS_METRIC)
+  }
+
+  /**
+   * Normalize a metric name to its IO diagnostic metric constant
```

Comment:
This comment seems to be generic. I believe we are performing normalization because we want to support variations in output rows such as `join output rows, number of output rows`

Can we specify the reason for this normalization in the comment?

Reply:
thanks, updated.